### PR TITLE
Update user stories

### PR DESCRIPTION
**Pull request by willowisp01**

### CODE REVIEW

**Review by yashma-sonara — APPROVED** [review]
User stories look good!

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -275,16 +275,17 @@ _{Explain here how the data archiving feature will be implemented}_
...
-| `* * *`  | user                                       | find a person by name          | locate details of persons without having to go through the entire list |
-| `* *`    | user                                       | hide private contact details   | minimize chance of someone else seeing them by accident                |
-| `*`      | user with many persons in the address book | sort persons by name           | locate a person easily                                                 |
+| Priority | As a …​                             | I want to …​                                       | So that I can…​                                    |
+|----------|-------------------------------------|----------------------------------------------------|----------------------------------------------------|
+| `* * *`  | Course Instructor who can type fast | enter new students’ information using command line | I can store the information efficiently            |
+| `* * *`  | Course Instructor                   | put students into groups                           | I can tell which group each of the students are in |
```
Comment (JiaXinEu) [inline docs/DeveloperGuide.md:281]:
```suggestion
| `* * *`  | Course Instructor who can type fast | enter new students’ information using command line | store the information efficiently            |
| `* * *`  | Course Instructor                   | put students into groups                           |  tell which group each of the students are in |
```

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -275,16 +275,17 @@ _{Explain here how the data archiving feature will be implemented}_
...
+| `* * *`  | Course Instructor                   | access student’s contact information               | communicate with them easily                       |
+| `* *`    | Course Instructor                   | delete student details on command line             | I can remove students no longer in class           |
+| `* * *`  | Seasoned Course Instructor          | view a single student’s details                    | to identify students of note                       |
```
Comment (JiaXinEu) [inline docs/DeveloperGuide.md:284]:
```suggestion
| `* *`    | Course Instructor                   | delete student details on command line             | remove students no longer in class           |
| `* * *`  | Seasoned Course Instructor          | view a single student’s details                    | identify students of note                       |
```

**Review by JiaXinEu — CHANGES_REQUESTED** [review]
Overall looks good! Might want to modify the sentence to match the heading of the table.

**Pull request merged**